packages/AsyncLoader/README.md

```diff
@@ -35,9 +35,13 @@ Component that fetches contents asynchronously and displays it when ready:
 `<async-loader></async-loader>`
 
 #### Attributes
-- `data-endpoint-url` (required): URL that should be fetched
+- `data-endpoint-url` (required if `data-event-endpoint-property-name` is not set): URL that should be fetched.
+Can be omitted `data-event-endpoint-property-name` is set.
```

Comment:
Ergänzen: "If both `data-endpoint-url` and `data-event-endpoint-property-name` are provided, `data-endpoint-url" will be preferred.` Im Gegenzug kannst "Can be omitted …" streichen, da's oben bereits vorkommt bei required.